github3/issues/issue.py

```diff
@@ -254,6 +258,18 @@ def labels(self, number=-1, etag=None):
         url = self._build_url('labels', base_url=self._api)
         return self._iter(int(number), url, Label, etag=etag)
 
+    @requires_auth
+    def lock(self, *args):
+        """Lock an issue.
+
+        :returns: bool
+        """
+        headers = {'Content-Length': 0}
```

Comment:
Ah yes, you're right. Sorry for misunderstanding.
I've tested it without setting those headers manually, and it works fine. I'll fix it up and rebase this branch.